### PR TITLE
pythonPackages.tflearn: 0.2.1 -> 0.3.2

### PR DESCRIPTION
**Pull request by jethrokuan**
###### Motivation for this change
ZHF #45960

first time contributor (not active user of this package either), just wanted to try my hand at this and help out.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -16810,19 +16810,24 @@ EOF
...
   tflearn = buildPythonPackage rec {
-    name = "tflearn-0.2.1";
+    version = "0.3.2";
+    name = "tflearn-${version}";
 
-    meta = {
-      description = "Deep learning library featuring a higher-level API for TensorFlow";
-      homepage    = "https://github.com/tflearn/tflearn";
-      license     = licenses.mit;
+    src = pkgs.fetchurl {
```
Comment (Mic92) [inline pkgs/top-level/python-packages.nix]: We are moving Python expressions out of `pkgs/top-level/python-packages.nix` into `pkgs/development/python-modules/<module>/default.nix`. 

Please move the expression there, and call it from `pkgs/top-level/python-packages.nix` using `callPackage ../development/python-modules/<package> { };`.

Comment (Mic92) [reply pkgs/top-level/python-packages.nix]: Example refactoring: https://github.com/NixOS/nixpkgs/pull/45763

**Comment by Mic92** [comment]
@GrahamcOfBorg build pythonPackages.tflearn python3Packages.tflearn

**Comment by Mic92** [comment]
tensorflow itself is broken too on aarch64, but this is out of scope of this pr.

**Comment by Mic92** [comment]
[detached HEAD c3796a7d1b6] pythonPackages.tflearn: 0.2.1 -> 0.3.2 (#46102)
 Author: Jethro Kuan <jethrokuan95@gmail.com>
 Date: Thu Sep 6 00:38:23 2018 +0800
 2 files changed, 25 insertions(+), 16 deletions(-)
 create mode 100644 pkgs/development/python-modules/tflearn/default.nix

**Comment by Mic92** [comment]
Thanks! Backport is above.

**Comment by jethrokuan** [comment]
Thanks for the review!

**Pull request merged**